keanu-python/tests/test_sampling.py

```diff
@@ -35,19 +40,90 @@ def test_sampling_returns_dict_of_list_of_ndarrays_for_vertices_in_sample_from(
         algo: Callable[[BayesNet], PosteriorSamplingAlgorithm], net: BayesNet) -> None:
     draws = 5
     sample_from = list(net.get_latent_vertices())
-    vertex_labels = [vertex.get_label() for vertex in sample_from]
-    samples = sample(net=net, sample_from=sample_from, sampling_algorithm=algo(net), draws=draws)
+    samples = sample(net=net, sample_from=sample_from, sampling_algorithm=algo(net), draws=draws, plot=True)
```

Comment:
why?

Reply:
Left over from debugging the plotting bug - will take it out